Resources/Prototypes/Recipes/Construction/Graphs/furniture/meatspike.yml

```
@@ -2,29 +2,117 @@
   id: MeatSpike
   start: start
   graph:
+
   - node: start
     actions:
-    - !type:DeleteEntity {}
+      - !type:DeleteEntity {}
     edges:
-    - to: MeatSpike
-      completed:
-      - !type:AdminLog # Needs a log for start of attempt in addition to the completion log
-        message: "Construction"
-        impact: High
-      - !type:SnapToGrid
-        southRotation: true
-      steps:
-      - material: Steel
-        amount: 15
-        doAfter: 2
-  - node: MeatSpike
+      # here to the assambly
+      - to: assembly
+        completed:
+          - !type:SnapToGrid
+            southRotation: true
+          - !type:AdminLog # Needs a log for start of attempt in addition to the completion log
+            message: "Construction"
+            impact: High
```

Incorrect indentation, see our codebase conventions on the dev wiki. There's more like this in the file.
```suggestion
        completed:
        - !type:SnapToGrid
          southRotation: true
        - !type:AdminLog # Needs a log for start of attempt in addition to the completion log
          message: "Construction"
          impact: High
```

---

Done. I think.